packages/strapi-plugin-content-type-builder/admin/src/containers/DataManagerProvider/utils/tests/cleanData.test.js

```diff
@@ -207,4 +208,19 @@ describe('CleanData utils', () => {
       ).toEqual(componentsToFormat.sort());
     });
   });
+
+  describe('sortContentType', () => {
+    it('should return sorted collection types array', () => {
+      const { sortedContentTypes } = expectedData;
+      const {
+        rawData: { contentTypesToSort },
+      } = rawData;
+
+      const actual = sortContentType(contentTypesToSort);
+      expect(actual.sort()).toEqual(sortedContentTypes.sort());
```

Comment:
If you could add an extra line before `expect` and between the `it`s it would be awesome.